### PR TITLE
Use children of toggle button for label

### LINKED ISSUE #40: [A11y 8/27] "unlabeled 0 toggle button" JAWS verbalization
On Windows with JAWS screen reader on Firefox and Chrome, I get the spoken feedback "unlabeled 0 toggle button" every time I access the toggles. Any idea why?

### PR DESCRIPTION
**Pull request by dfreedm**
Fixes #40

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -71,33 +67,33 @@
       <div>
         <h4>Enabled</h4>
         <div class="horizontal-section">
-          <div class="line"><paper-toggle-button></paper-toggle-button><span>Oxygen</span></div>
-          <div class="line"><paper-toggle-button></paper-toggle-button><span>Carbon</span></div>
-          <div class="line"><paper-toggle-button checked></paper-toggle-button><span>Hydrogen</span></div>
-          <div class="line"><paper-toggle-button checked></paper-toggle-button><span>Nitrogen</span></div>
-          <div class="line"><paper-toggle-button checked></paper-toggle-button><span>Calcium</span></div>
+          <div class="line"><paper-toggle-button><span>Oxygen</span></paper-toggle-button></div>
```
Comment (notwaldorf) [inline demo/index.html]: I don't think these need to be in spans anymore

File: paper-toggle-button.html [hunk 2]
```diff
@@ -153,6 +154,17 @@
...
+
+      #toggleLabel {
+        position: relative;
+        top: -2px;
+        display: inline-block;
+        vertical-align: middle;
+        margin-left: 10px;
+        white-space: normal;
+        pointer-events: none;
+        color: var(--paper-toggle-button-label-color, --primary-text-color);
```
Comment (notwaldorf) [inline paper-toggle-button.html:153]: This needs to be documented :)

Comment (dfreedm) [inline paper-toggle-button.html:153]: It is now

File: demo/index.html [hunk 3]
```diff
@@ -71,33 +67,33 @@
       <div>
         <h4>Enabled</h4>
         <div class="horizontal-section">
-          <div class="line"><paper-toggle-button></paper-toggle-button><span>Oxygen</span></div>
-          <div class="line"><paper-toggle-button></paper-toggle-button><span>Carbon</span></div>
-          <div class="line"><paper-toggle-button checked></paper-toggle-button><span>Hydrogen</span></div>
-          <div class="line"><paper-toggle-button checked></paper-toggle-button><span>Nitrogen</span></div>
-          <div class="line"><paper-toggle-button checked></paper-toggle-button><span>Calcium</span></div>
+          <div class="line"><paper-toggle-button>Oxygen</paper-toggle-button></div>
```
Comment (notwaldorf) [inline demo/index.html]: i think you can remove the `div` now

**Comment by notwaldorf** [comment]
one minor demo comment, but otherwise :+1:

**Pull request merged**